packages/muze-axis/src/cartesian-axis/band-axis.js

```diff
@@ -63,6 +58,18 @@ export default class BandAxis extends SimpleAxis {
         return this;
     }
 
+    /**
+     * This method is used to set the padding between plots.
+     *
+     * @param {number} padding padding value supplied from config.
+     * @memberof BandAxis
+     */
+    updateAxisPadding (padding) {
+        if (typeof padding === 'number') {
```

Comment:
Dont do this check here, have a sanitization function.
The sanitization function in config should not accept padding if it's not a number. Also, what happens if the padding is negative?

Reply:
Moved this to config sanitization and added check for padding being negative